openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb

```diff
@@ -2591,8 +2590,8 @@ def self.utility_bills_rates_section(model, sqlFile, runner, name_only = false)
     # create table
     energy_cost_table = {}
     energy_cost_table[:title] = 'Energy Cost Summary'
-    energy_cost_table[:header] = ['', 'Process Subtotal', 'Total Energy Cost']
-    energy_cost_table[:units] = ['', '$', '$']
+    energy_cost_table[:header] = ['', 'Total Energy Cost'] # skipping Process Subtotal
```

Comment:
This is not related to 8.6. @asparke2 just wanted to hide Process Subtotal which was confusing.